packages/sdk/src/client.ts

```diff
@@ -84,7 +82,8 @@ export class RecallClient {
     }
     const chain = this.publicClient.chain;
     if (!chain) throw new Error("missing chain in provided client");
-    this.network = config.network ?? Network.fromChain(chain);
+    this.subnetId = SubnetId.fromChain(chain);
```

Comment:
need for cross-chain flows like deposits and withdrawals. (this used to be part of the `Network`, which we've removed.)